industrial_ci/src/builders/catkin_tools.sh

```diff
@@ -59,7 +59,7 @@ function builder_run_tests {
         opts+=(-i)
     fi
     _append_job_opts opts PARALLEL_TESTS 1
-    ici_cmd ici_exec_in_workspace "$extend" "$ws" catkin build --catkin-make-args run_tests -- "${opts[@]}" --no-status
+    ici_cmd ici_exec_in_workspace "$extend" "$ws" catkin run_tests "${opts[@]}" --no-status
```

Comment:
There was a reason for using this instead of run_tests, but I can't recall it at the moment.
I guess it did not exists for indigo..

Reply:
I got introduced in https://github.com/ros-industrial/industrial_ci/pull/255, so I it should not be needed anymore.
I will run a local test against indigo to be sure.